Documentation/Concepts/Index.rst

```diff
@@ -30,6 +30,14 @@ TYPO3 Concepts
         ..  card-footer:: :ref:`Quick introduction to Fluid <fluid-templates>`
             :button-style: btn btn-secondary stretched-link
 
+    ..  card:: TCA
+
+        The Table Configuration Array (TCA) defines how database records are
+        displayed and managed in the TYPO3 backend.
```

Comment:
TCA effects more then just the backend. It also affects Models in Extbase, data import and manipulation, and the new records objejects